src/rpcblockchain.cpp

```diff
@@ -323,7 +323,8 @@ UniValue getblockheader(const UniValue& params, bool fHelp)
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
             "  \"previousblockhash\" : \"hash\",  (string) The hash of the previous block\n"
-            "  \"nextblockhash\" : \"hash\"       (string) The hash of the next block\n"
+            "  \"nextblockhash\" : \"hash\",      (string) The hash of the next block\n"
+            "  \"chainwork\" : \"0000...1f3\"     (string) Expected number of hashes required to produce the current chain (in hex)\n"
```

Comment:
This message is based on [@sipa's description of chainwork](https://bitcoin.stackexchange.com/questions/26869/what-is-chainwork) and [was vetted previously by @petertodd](https://github.com/bitcoin/bitcoin/pull/7194#discussion_r47325775). You may want to close the PR you reference above; it was opened after this one and doesn't include tests.